Framework/src/Check.cxx

```diff
@@ -122,6 +122,13 @@ void Check::initConfig(std::string checkName)
   mCheckConfig.detectorName = checkConfig.get<std::string>("detectorName", "DET");
   mLatestQuality->setDetectorName(mCheckConfig.detectorName);
 
+  // Determine whether we can beautify
+  // See QC-299 for details
+  if (numberOfTaskSources > 1) {
+    mBeautify = false;
+    ILOG(Warning) << "Beautification disabled because more than one source is used in this Check (" << mCheckConfig.checkName << ")" << ENDM;
```

Comment:
I am not exactly sure, but `Check::initConfig()` is probably invoked even during workflow generation. If so, then this warning will be visible in each device in a workflow, so we might unnecessarily pollute the logs. If that happens indeed, could you put this to `Check::init()`, which AFAIK should be invoked only by an initializing CheckRunner.

Reply:
good call, changing it

Reply:
I have also moved the rest of the logging from the function. I understand now why we were getting this info multiple times in the console.

Reply:
Nice, thanks!